bazel/repositories.bzl

```diff
@@ -80,6 +80,7 @@ def _envoy_repo_impl(repository_ctx):
     repo_path = repository_ctx.path(repository_ctx.attr.envoy_root).dirname
     version = repository_ctx.read(repo_path.get_child("VERSION")).strip()
     repository_ctx.file("version.bzl", "VERSION = '%s'" % version)
+    repository_ctx.file("path.bzl", "PATH = '%s'" % repo_path)
```

Comment:
@lizan i have a feeling that this change is related to the objection you raised previously - but hoping not

as we have moved a lot of the tooling to python packages and the pytooling repo - its handy to be able to pass the pass as arg (rather than importing it in python) - hence this change

Reply:
This is fine. I'm ok with it as long as it is kept in a separated repository so it doesn't pollute cache.